flink-connector-mongodb/src/test/java/org/apache/flink/connector/mongodb/testutils/MongoTestUtil.java

```diff
@@ -62,7 +62,7 @@ private MongoTestUtil() {}
      * @return configured MongoDB container
      */
     public static MongoDBContainer createMongoDBContainer(Logger logger) {
-        return new MongoDBContainer(DockerImageName.parse(MONGO_4_0))
+        return new MongoDBContainer(mongoDockerImageName())
                 .withLogConsumer(new Slf4jLogConsumer(logger));
     }
```

Comment:
Minor: What about construct `MongoDBContainer` with a generic `Consumer` instead of a Logger? Maybe we need to construct a container and capture its output to something like `ToStringConsumer()`.

Reply:
Since subsequent configuration items can still be set using chained calls after constructing `MongoDBContainer`, the `Logger` parameter is removed here to gain greater flexibility.